app/src/main/res/values/strings.xml

```diff
@@ -859,6 +873,9 @@
     <string name="common_weather_text_frost">Frost</string>
     <string name="common_weather_text_squall">Squall</string>
     <string name="common_weather_text_tornado">Tornado</string>
+    <string name="common_weather_text_blowing_snow">Blowing snow</string>
+    <string name="common_weather_text_drifting_snow">Drifting snow</string>
+    <string name="common_weather_text_glaze">Glaze</string>
```

Comment:
Same thing as `precipitation_ice`? At least given the context I see it used

Reply:
I was following the legends [here](https://www.ilmateenistus.ee/ilm/prognoosid/legend/?lang=en).

I don't think these three mean the same thing as `precipitation_ice`. Here are NWS's definitions of the three:
- [Blowing snow](https://forecast.weather.gov/glossary.php?word=blowing%20snow)
- [Drifting snow](https://forecast.weather.gov/glossary.php?word=drifting%20snow)
- [Glaze](https://forecast.weather.gov/glossary.php?word=glaze)

But if you meant just for "Glaze", then I think "Ice" could work. However, is it "safe" or "proper" to appropriate `precipitation_ice` for this purpose? We didn't appropriate `precipitation_rain` for other instances of rain, nor `precipitation_thunderstorm` for other instances of thunderstorm.

Reply:
I see, "Ice" is more the form of the water, while "Glaze" is more the slippery condition that resulted from the accumulation of ice. I need to fix my French translation :p

Reply:
Then I'll just keep these three lines.